Tests/BUILD

```diff
@@ -0,0 +1,87 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+load(
+    "@build_bazel_rules_apple//apple:ios.bzl",
+    "ios_unit_test",
+    "ios_application"
+)
+
+load(
+    "@build_bazel_rules_apple//apple:macos.bzl",
+    "macos_unit_test"
+)
+
+load(
+    "@build_bazel_rules_apple//apple:tvos.bzl",
+    "tvos_unit_test",
+    "tvos_application"
+)
+
+MINIMUM_IOS_VERSION = "8.0"
+
+MINIMUM_MACOS_VERSION = "10.9"
+
+MINIMUM_TVOS_VERSION = "9.0"
+
+swift_library(
+    name = "KronosTests",
+    srcs = glob(["KronosTests/*.swift"]),
+    deps = ["//:Kronos"]
+)
+
+genrule(
+    name = "AppDelegate",
+    outs = ["AppDelegate.swift"],
+    cmd = """
+      echo "import UIKit\n\n@UIApplicationMain\nclass AppDelegate: UIResponder, UIApplicationDelegate {\nvar window: UIWindow?\n}" > $@
+    """
+)
+
+genrule(
+    name = "ProcessedInfoPlist",
+    srcs = ["//Resources:InfoPlist"],
+    outs = ["GeneratedInfo.plist"],
+    cmd = "/usr/libexec/PlistBuddy -x $< -c \"Set :CFBundleExecutable Kronos\" -c \"Set :CFBundleIdentifier com.Lyft.Kronos\" -c \"Set :CFBundleVersion 1.0.0\" -c \"Print\" > $@"
+)
+
+swift_library(
+    name = "lib",
+    srcs = ["AppDelegate.swift"],
+)
+
+ios_application(
+    name = "iOSTestApp",
```

Comment:
Why do we need a test host?

Reply:
I could not get these tests to pass locally without a host

Reply:
ah yea probably the weirdness around running without booted simulators